Misc/NEWS.d/next/Library/2022-08-22-13-54-20.gh-issue-96175.bH7zGU.rst

```diff
@@ -0,0 +1 @@
+Fix unused ``localName`` parameter in ``Attr`` minidom node.
```

Comment:
```suggestion
Fix unused ``localName`` parameter in the ``Attr`` class in :mod:`xml.dom.minidom`.
```
This will link to https://docs.python.org/3.10/library/xml.dom.minidom.html. (We don't seem to have docs for the `Attr` class specifically, or I'd have suggested ```:class:`xml.dom.minidom.Attr` ```.)